praxis.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.bindir = 'bin'
   spec.executables << 'praxis'
 
-  spec.add_dependency 'rack', '~> 1'
+  spec.add_dependency 'rack', '>= 1'
```

Comment:
Does is make sense to make this `>= 1 < 3` so that this doesn't break with Rack 3?